managed/services/telemetry/config.default.yml

```diff
@@ -245,3 +245,29 @@ telemetry:
     data:
       - metric_name: "pmm_client_node_type"
         column: "node_type"
+
+  # DBaaS
+  - id: DBaaSServicesCount
+    source: PMMDB_SELECT
+    query: select count(service_name) AS count from services where service_name ~ concat('default-', cluster,'-(pxc|rs0)-\d+')
+    summary: "DBaaS Services Count"
+    data:
+      - metric_name: "dbaas_services_count"
+        column: "count"
+
+  - id: DBaaSClustersCount
+    source: PMMDB_SELECT
+    query: select count(distinct cluster) AS count from services where service_name ~ concat('default-', cluster,'-(pxc|rs0)-\d+')
+    summary: "DBaaS Clusters Count"
+    data:
+      - metric_name: "dbaas_clusters_count"
+        column: "count"
```

Comment:
what is the difference between these 2? as I understand from PMM point of view cluster == service, isn't it?

Reply:
No, one cluster might have multiple services, one pod - one service